contact.css

```diff
@@ -0,0 +1,253 @@
+body {
+    margin: 0px; 
+    color: black;
+    background-image: url("./images/KH_backgr_a5_1.jpg");
+    text-align: center;
+  }
+  h1 {
+    font-size: 55px;
+    font-family: 'Archivo Black', sans-serif;
+    padding: 50px 10px;
+    margin: 15px 10px;
+    text-align: center;
+    color: #A39161;
+    letter-spacing: 5px;
+  }
+  p {
+    font-size: 18px;
+    font-family: Nunito;
+    letter-spacing: 2px;
+    text-align: justify;
+    padding: 15px 0;
+    margin: 15px 0;
+    line-height: 1cm;
+ } 
+  
+  /*** Header ***/
+  header {
+    height: 100px;
+    display: flex;
+    align-items: center;
+    padding: 50px 16px;
+ }
+  
+  .hamburger{
+    display: flex;
+    height: 30px;
+    width: 30px;
+    flex-direction: column;
+    justify-content: space-between;
+    position: absolute;
+    top: 80px;
+    right: 90px;  
+  }
+  
+  .hamburger div {
+    width: 30px;
+    height: 5px;
+    background:black;
+    display: flex;
+   margin-left: 50px;  
+  }
+  
+
+  nav {
+    display: none;
+    font-family: Nunito;
+    margin: 0 5px;
+    font-size: 18px;
+  }
+  
+  nav a {
+    display: inline-block;
+    padding: 5px;
+    text-decoration: none;
+    border-radius: 5px;
+    color: black;
+  }
+  
+  nav a:hover {
+    background-color: white;
+    color: hotpink;
+  }
+  
+  header img {
+    width: 200px;
+    margin: 15px;  
+  }
+/*** Body***/
+  .container {
+    display: block;
+    width: 420px;
+    margin-left: auto;
+    margin-right: auto;
+   
+}
+
+.container img {
+    width: 420px;  
+}
+.form {
+  display: block;
+}
+
+form {
+  display: inline-block;
+  text-align: left;
+  background-color: white;
+}
+
+span {
+  color: hotpink;
+  font-weight: bold;
+}
+
+input {
+  border: 1px solid #d8d8d8;
+  padding: 10px 15px;
+  margin: 0 0 20px;
+  width: 420px;
+  height: 40px;
+  display: block;
+  box-sizing: border-box;
+  border-radius: 5px;
+  font-family: helvetica; 
+  font-size: 15px;
+  color: #808080;
+}
+
+label {
+  font-size: 13px;
+  display: block;
+  font-family:  helvetica;
+  font-weight: bold;
+  color: #51595d;
+  margin-bottom: 10px;
+  margin-top: 20px; 
+}
+
+textarea {
```

Comment:
If you add `resize: none;` you will prevent the user from resizing your textarea :)